HopsanGenerator/src/generators/HopsanGeneratorBase.cpp

```diff
@@ -148,25 +148,34 @@ QString HopsanGeneratorBase::generateSourceCodefromComponentSpec(ComponentSpecif
     QString varDeclarations;
     for(int i=0; i<comp.parNames.size(); ++i)
     {
-        varDeclarations.append("        double "+comp.parNames[i]+";\n");
+        if(!varDeclarations.isEmpty()) {
+            varDeclarations.append("        ");
```

Comment:
hard-coding indentation is bizarre, but we can fix this in "the future", good for now.
I have written some automation for this in some place, but  I dont remember where, and probably not applicable to this code

Reply:
Sure, we can add a setting for indentation size later. It should then affect both the generated code and tabbing in the text editor.